tasks/concurrent.js

```diff
@@ -14,6 +14,14 @@ module.exports = function (grunt) {
 		var tasks = this.data.tasks || this.data;
 		var flags = grunt.option.flags();
 
+		if (flags.indexOf('--no-color') === -1 &&
+			flags.indexOf('--no-colors') === -1 &&
+			flags.indexOf('--color=false') === -1) {
+			// append the flag so that support-colors won't return false
+			// see issue #70 for details
+			flags.push('--color');
+		}
```

Comment:
This is adding the --color flag as expected. But it doesn't seem to do anything. debug js is still returning false for tty.isatty(2) (whereas on the older version of grunt-concurrent somehow it works). https://github.com/sindresorhus/grunt-concurrent/issues/86